src/Pages/PageNotFoundComponent.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from "react";
+import "../Stylesheets/App.css";
+import { Analytics } from "aws-amplify";
+import StartPage from "./StartPage";
+
+class PageNotFoundComponent extends Component {
+  render() {
+    Analytics.record("appRender");
+    return (
+      <div className="content">
+        <h1 className="pageHeader">Oh Hello</h1>
+        <p className="paragraphText">
+          these are not the droids you're looking for
```

Comment:
nit: Your single quote is messin' with the github parser.

You could codify it! 
```jsx
{`these are not the droids you're looking for`}
```